docs/source/losses.rst

```diff
@@ -91,6 +91,15 @@ Registration Losses
 .. autoclass:: GlobalMutualInformationLoss
     :members:
 
+Reconstruction Losses
+----------------------
+
+`SSIMLoss`
+~~~~~~~~~~~
```

Comment:
Please try to ensure all the length of "~~~~", "-----" match the corresponding string content.

Thanks.